docs/11/index.md

```diff
@@ -1174,37 +1322,70 @@ Let's gain a brief understanding of the first theorems and axioms.
 We are entering the foundations of the foundations - the very basement -
 of the "typical" mathematics of classical logic and ZFC set theory.
 
-The first theorem we have is `a1ii`.
-It has two hypotheses and one conclusion, and
-all the statements begin with `|-` meaning "this is true".
-Theorem `a1ii` can be interpreted as saying that if some `ph` is true,
-and some `ps` is true, then that `ph` is true.
-This simply lets us restate truths; this is only useful in
-special technical situations, but it's hard to argue with the conclusion.
-
-Theorem `idi` is similar; it says that if some `ph` is true, then that
-`ph` is true.
-This also lets us restate truths; this is again only useful in
-special technical situations, but it's hard to argue with the conclusion.
-This theorem was contributed by Alan Sare, which you can see by clicking
-on the ">" next to the name. The names of people who formalized and
-proposed various statements are included in their description.
+Assertion 1 is the theorem `idi`.
+
+<!-- TBD: Put image of idi here, once it's renumbered -->
+
+In `idi` all the statements begin with `|-` meaning "this is true".
+Theorem `idi` says the following: if we assume that some
+statement `ph` is true, then we can conclude that the statement `ph` is true.
+This theorem doesn't let us conclude anything new, so it's normally not useful;
+it's only useful in special technical situations.
+Let's see its description.
+
+> Click on the symbol "&gt;" to the right of the name **idi**.
+
+Every axiom or theorem in `set.mm` has a description.
+In this example, you can see that theorem `idi` was contributed by Alan Sare.
+The names of people who formalized and proposed various statements
+are recorded for all time in these descriptions.
+
+☞ We hope to see *your* name in future database descriptions! ☜
+
+You can also see that the description of theorem `idi` says
+"(New usage is discouraged.)".
+This tells tools (like metamath-lamp) to *not* use this
+theorem unless the human specially directs the tool to use it
+(e.g., by manually entering it as a justification).
+Theorem `idi` doesn't let us determine new truths, so we don't
+normally want to use it.
+Metamath-lamp will not, by default, mention this as an option
+when you're trying to prove something (since it's normally not useful).
+That said, this statement *is* true, and you can use it if you want to.
+
+Assertion 2 is the theorem is `a1ii`.
+It has two hypotheses and one conclusion.
+All the statements begin with `|-` which again means "this is true".
+Theorem `a1ii` can be interpreted as saying that if we assume
+some `ph` is true,
+and we assume some `ps` is true, then we can conclude that `ph` is true.
+This simply lets us restate one of two accepted truths.
+Again, this is only useful in special technical situations,
+but it's hard to argue with the conclusion.
+This theorem is also discouraged, so again, metamath-lamp will normally
+not try to use it.
```

Comment:
`This theorem is also discouraged, so again, metamath-lamp will normally not try to use it.`

Actually mm-lamp doesn't exclude discouraged assertions. Probably it will be better to rephrase this statement:

`This theorem is also discouraged, so again, normally you should not use it.`